src/components/dialog/bl-dialog.test.ts

```diff
@@ -1,14 +1,8 @@
-import {
-  assert,
-  fixture,
-  html,
-  oneEvent,
-  expect,
-  fixtureCleanup,
-} from '@open-wc/testing';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
```

Comment:
Why do we need those?

Reply:
We wanted to mock `window.HTMLDialogElement` object in tests as true or false value but typescript don't let this operation. If you have any idea for this it would be very help .